docs/configure-upstream-collector.md

```diff
@@ -2,8 +2,8 @@
 
 You can build and configure a [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or extend the [OpenTelemetry Collector Contrib ](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution to collect logs and metrics and send them to Elastic Observability.
 
-For a more seamless experience, use the Elastic Distribution for the OpenTelemetry collector.
-Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic OTel collector.
+For a more seamless experience, use the Elastic Distribution for the OpenTelemetry Collector.

```

Comment:
We want to make this as detailed as possible. Lets elaborate more on how to use [ocb](https://opentelemetry.io/docs/collector/custom-collector/) and what components should be included and minimize sending the user to external links to get through to doing it.

We should send users to external links only if they want to go deeper or do something custom/advanced.

Reply:
This is in progress in [this issue](https://github.com/orgs/elastic/projects/649/views/43?pane=issue&itemId=73752307). I'll pull this content out of the issue into a new PR so we can finalize the details in GH. I don't think that should stop us from merging this PR though.